packages/kit/src/states/jotai/contexts/swap/actions.ts

```
@@ -1753,6 +1786,17 @@ class ContentJotaiActionsSwap extends ContextJotaiActionsBase {
               void this.resetSwapTokenData.call(set, ESwapDirectionType.TO);
             }
           }
+          const fromLimitTokenDefault = fromNetworkDefault?.limitFromToken;
+          if (
+            fromToken &&
+            fromToken.isNative &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)